docs/cell_census_storage_and_release_policy.md

```diff
@@ -1,52 +1,73 @@
+* **Cell Census source h5ads**: the set of h5ad files used for create a Cell Census build.
+* **Cell Census release**: a Cell Census build that is officially and publicly hosted online.
+* **Cell Census release `tag`**:  a label for a Cell Census release, it MUST be a string of printable ASCII characters.
 
-`s3://cellxgene-data-public/`
+## Cell Census release policy
 
-All contents of a Cell Census build data MUST be deposited in a folder named `cell-census`: 
+### Long-term supported Cell Census release
 
-`./cell-census/`
+* It MUST be created and made available for public access every six months. 
+* Its date of publication MUST be used as the `release_date` (see [below](#Cell-Census-release-information)). 
+* All data from CELLxGENE Discover hosted at the `release_date` that meets the requirements [according to the schema](https://github.com/chanzuckerberg/cell-census/blob/main/docs/cell_census_schema.md#data-included) MUST be included in it.
+* It MUST be available for public access for at least 5 years upon `release_date`.
 
-This folder MUST contain all public builds and releases defined as:
+### Weekly Cell Census release (latest)
 
-* Census "build" is a SOMA collection / container containing cell data.
-* Census "RELEASE" is a "build" which has been officially blessed and released in a formal manner.
-* A "build" is named with a `tag`, which is a string of printable ASCII (eg, "foo", "2022-11-01", "1-rc2.9"):
-   `./cell-census/[tag]/`
-* a RELEASE is named with a tag following the conventional format `release-[counter]`:
-   `./cell-census/release-[counter]/`
+* It MUST be created and made available for public access every week on Mondays except for CZI holidays.
+* Its date of publication MUST be used as the `release_date` (see [below](#Cell-Census-release-information)). 
+* All data from CELLxGENE Discover hosted at the `release_date` that meets the requirements [according to the schema](https://github.com/chanzuckerberg/cell-census/blob/main/docs/cell_census_schema.md#data-included) MUST be included in it.
+* It MUST be deleted the following Monday and after the next weekly Cell Census release is published.
 
-The Cell Census SOMA-related files MUST be deposited within the release folder on a folder named `soma`:
 
-`./cell-census/release-[counter]/soma/`
+## Cell Census data storage policy
 
-All h5ads used to create the Cell Census MUST be copied within the release folder into a folder named `h5ads`:	
-`./cell-census/release-[counter]/h5ads/`
+The following S3 bucket MUST be used as the root to store Cell Census data:
 
-Any dataset changes, additions, or deletions per release MUST be documented in the following human-readable changelog file name `changelog.txt`:
+`s3://cellxgene-data-public/`
 
-`./cell-census/changelog.txt`
+Cell Census data MUST deposited under a folder named `cell-census` of the root S3 bucket:
```

Comment:
```suggestion
Cell Census data MUST be deposited under a folder named `cell-census` of the root S3 bucket:
```

Technically "folder" is "prefix", but this is interpretable as is.

Could combine the above two bullets